Tests/govuk_ios/govuk_ios_unit_tests/Specs/ViewModels/SettingsViewModelTests.swift

```diff
@@ -38,9 +43,13 @@ struct SettingsViewModelTests {
 
         let aboutTheAppSection = sut.listContent[0]
         #expect(aboutTheAppSection.heading?.title == "About the app")
+
         let helpAndFeedbackRow = try #require(aboutTheAppSection.rows.first as? LinkRow)
+        let url = "https://www.gov.uk/contact/govuk-app?app_version=123%20(456)&phone=Apple%20iPhone16,2%2018.1"
```

Comment:
rename to expectedUrl, it's a bit clearer

Reply:
Yeah, I tried to keep line length down with just `url` but swapped to `expectedUrl`